x/slashing/spec/02_state.md

```diff
@@ -12,7 +12,7 @@ long as it contains precommits from +2/3 of total voting power.
 
 Proposers are incentivized to include precommits from all validators in the Tendermint `LastCommitInfo`
 by receiving additional fees proportional to the difference between the voting
-power included in the `LastCommitInfo` and +2/3 (see [fee distribution](https://github.com/cosmos/cosmos-sdk/blob/master/docs/spec/fee_distribution)).
+power included in the `LastCommitInfo` and +2/3 (see [fee distribution](x/distribution/spec/03_begin_block.md)).
```

Comment:
```suggestion
power included in the `LastCommitInfo` and +2/3 (see [fee distribution](./03_begin_block.md)).
```

Reply:
I don't think that would work, the `03_begin_block.md` file in question is in the distribution doc, this change is in the slashing docs. Different path.

Reply:
@fedekunze  ^^^